contracts/src/v0.8/functions/dev/v1_X/ocr/OCR2Base.sol

```diff
@@ -342,10 +348,9 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
         if (o.role != Role.Signer) revert ReportInvalid("address not authorized to sign");
         if (signed[o.index] != address(0)) revert ReportInvalid("non-unique signature");
         signed[o.index] = signer;
-        signerCount += 1;
       }
```

Comment:
looks like signerCount was never actually used? Well, I guess we saved a bit of gas :)